test/func/sproxyd_utils.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) 2014 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import with_statement
+import os
+import platform
+import subprocess32
+import urllib
+
+
+ring_driver_conf_tmpl = """"ring_driver:0": {
+        "by_path_service_id": "%(by_path_service_id)s",
+        "by_path_cos": 0,
+        "alias": "chord_path",
+        "bstraplist": "%(bootstraplist)s",
+        "by_path_enabled": true,
+        "deferred_deletes_enabled_by_policy": false,
+        "deferred_deletes_enabled_by_request": false,
+        "deferred_writes_enabled_by_policy": false,
+        "deferred_writes_enabled_by_request": false,
+        "type": "chord"
+     }"""
+
+sproxyd_conf_tmpl = """{
+    "general": {
+        "ring": "MyRing",
+        "conn_max": 10000,
+        "conn_max_reuse": 100000,
+        "consistent_reads": true,
+        "consistent_writes": true,
+        "max_proc_fd": 40960,
+        "port": %(fcgi_port)s,
+        "split_chunk_size": 33554432,
+        "split_control_by_request": false,
+        "split_enabled": true,
+        "split_gc_cos": 2,
+        "split_memory_limit": 671088640,
```

Comment:
I meant here, do we have to tune the number of workers and the memory settings for these tests ?